imageSequence.ml

```diff
@@ -0,0 +1,101 @@
+(*
+ * This file is part of Imagelib.
+ *
+ * Imagelib is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Imabelib is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Imabelib.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright (C) 2014 Rodolphe Lepigre.
+ *)
+
+open Image
+open ImageUtil
+
+let only_once ich read_chunks ctype =
+  if List.mem ctype !read_chunks
+  then begin
+    let msg = Printf.sprintf
+        "Chunk %s should not appear more than once..." ctype
```

Comment:
`%S` ?

Reply:
So far both `ImagePNG` and `ImageJPG` use printable strings to represent headers, but sure.